Monika After Story/game/zz_selector.rpy

```diff
@@ -2754,6 +2754,22 @@ label mas_selector_sidebar_select_clothes(items, preview_selections=True, only_u
 # [MONSEL]
 
 #### Begin monika clothes topics
+init 200 python:
+    # list of event clothes
+    event_clothes = [
+        mas_selspr.CLOTH_SEL_MAP["sundress_white"],
+        mas_selspr.CLOTH_SEL_MAP["santa"],
+        mas_selspr.CLOTH_SEL_MAP["rin"],
+        mas_selspr.CLOTH_SEL_MAP["marisa"]
+    ]
+
+    # need to get a list of clothes that have been gifted
+    # so we will get a list of all clothes and then remove the event_clothes
+    gifted_clothes = mas_selspr.filter_clothes(True)
+
+    for i in event_clothes:
```

Comment:
can you use a variable name that is more descriptive of what `i` is? Maybe `cloth_sel_obj` instead